<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1281</width>
    <height>642</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFontComboBox" name="fontComboBox">
    <property name="geometry">
     <rect>
      <x>980</x>
      <y>550</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="SBFontSize">
    <property name="geometry">
     <rect>
      <x>1180</x>
      <y>550</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="value">
     <number>11</number>
    </property>
   </widget>
   <widget class="QScrollArea" name="scrollArea">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>1281</width>
      <height>501</height>
     </rect>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>1277</width>
       <height>497</height>
      </rect>
     </property>
     <widget class="QTextEdit" name="textEdit">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1281</width>
        <height>491</height>
       </rect>
      </property>
     </widget>
     <widget class="QScrollBar" name="verticalScrollBar">
      <property name="geometry">
       <rect>
        <x>1250</x>
        <y>10</y>
        <width>16</width>
        <height>160</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
     <widget class="QScrollBar" name="horizontalScrollBar">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>430</y>
        <width>160</width>
        <height>16</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPlainTextEdit" name="PTEFileName">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>550</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="PBBold">
    <property name="geometry">
     <rect>
      <x>1000</x>
      <y>520</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Bold</string>
    </property>
   </widget>
   <widget class="QPushButton" name="PBItalic">
    <property name="geometry">
     <rect>
      <x>1070</x>
      <y>520</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Italic</string>
    </property>
   </widget>
   <widget class="QPushButton" name="PBUnderline">
    <property name="geometry">
     <rect>
      <x>1150</x>
      <y>520</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Underline</string>
    </property>
   </widget>
   <widget class="QComboBox" name="doctypeComboBox">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>550</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>.txt</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>.dxcs</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>.doc</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="labelFileName">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>530</y>
      <width>91</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>File Name:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="PBSave">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>550</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Save</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="RBAutoSave">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>560</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Auto-Save</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1281</width>
     <height>28</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuNote">
    <property name="title">
     <string>Note</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuBrowse">
    <property name="title">
     <string>Browse</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuConnect">
    <property name="title">
     <string>Connect</string>
    </property>
   </widget>
   <addaction name="menuNote"/>
   <addaction name="menuBrowse"/>
   <addaction name="menuConnect"/>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
